lug/worker-script/ubuntu-rsync.sh

```diff
@@ -0,0 +1,35 @@
+#/bin/bash
```

Comment:
Seems that this file is exactly the same as `debian-rsync.sh`. You may delete one of it since we could configure source and dir from environment variable.

Reply:
Got it, thanks!